experimental/saitypesextensions.h

```diff
@@ -180,10 +184,32 @@ typedef enum _sai_dash_flow_enabled_key_t
 
 /**
  * @brief Defines a list of enums for dash_flow_action
+ *
+ * @flags strict
  */
 typedef enum _sai_dash_flow_action_t
 {
-    SAI_DASH_FLOW_ACTION_NONE,
+    SAI_DASH_FLOW_ACTION_NONE = 0,
+
+    SAI_DASH_FLOW_ACTION_ENCAP_U0 = 1,
+
+    SAI_DASH_FLOW_ACTION_ENCAP_U1 = 2,
+
+    SAI_DASH_FLOW_ACTION_SET_SMAC = 4,
+
+    SAI_DASH_FLOW_ACTION_SET_DMAC = 8,
+
+    SAI_DASH_FLOW_ACTION_SNAT = 16,
+
+    SAI_DASH_FLOW_ACTION_DNAT = 32,
+
+    SAI_DASH_FLOW_ACTION_NAT46 = 64,
+
+    SAI_DASH_FLOW_ACTION_NAT64 = 128,
+
+    SAI_DASH_FLOW_ACTION_SNAT_PORT = 256,
+
+    SAI_DASH_FLOW_ACTION_DNAT_PORT = 512,
```

Comment:
mayeb use arrows << like other to easy spot which bit is turned on?

Reply:
agree

Reply:
fixed